### PR TITLE
Fixes #611, Auto correct shown in the first page only

### LINKED ISSUE #611: Only show "Did you mean:" feature on page 1.
**Actual Behaviour**

When a user, goes to page 2 - the autocorrect feature is still there. 

**Expected Behaviour**

The autocorrect feature should be shown only on page 1 of the results page. And not on other pages.
Follow market leader design.

### PR DESCRIPTION
**Pull request by Marauderer97**
Fixes issue #611 

Changes: Auto correct is shown in the first page only, hidden from rest.

Demo Link: [Here](https://marauderer97.github.io/susper.com/)

Screenshots for the change: 

(Notice the URL)
image
image

### CODE REVIEW

File: src/app/auto-correct/auto-correct.component.ts [hunk 1]
```diff
@@ -12,13 +12,24 @@ import * as fromRoot from '../reducers';
 
 export class AutoCorrectComponent implements OnInit {
   query$: any;
+  wholequery$: any;
   suggestion: any;
   sugflag: boolean;
+  isPage1: boolean;
```
Comment (harshit98) [inline src/app/auto-correct/auto-correct.component.ts]: Would be better if you use `isStartPage`. Don't use any number in the variable.

Comment (nikhilrayaprolu) [inline src/app/auto-correct/auto-correct.component.ts]: why do we need another variable here. isn't it enough to check `presentPage===1`

**Comment by Marauderer97** [comment]
@nikhilrayaprolu How can I access presentPage in the autocorrect component?

**Comment by Marauderer97** [comment]
@harshit98 Please check, the requested changes have been made

**Comment by Marauderer97** [comment]
@nikhilrayaprolu Please check now :+1:

**Comment by Marauderer97** [comment]
@mariobehling I took this screenshot from my phone:
image

**Comment by mariobehling** [comment]
Ok, good that it is working for you. I hope you see the issues with the mobile display though and help to fix them. Let's focus on what we have and ensure it works before following up on more experimental features. I feel I am repeating this a bit too often now. The GSoC program started over two months ago. We have some nice features, but some basics e.g. the site on a phone, are still not working. Please help!

**Pull request merged**